components/shopify_partner/README.md

```diff
@@ -0,0 +1,18 @@
+# Shopify Partner API Integration
+
+## Configure API Key
+
+In order to authorize Pipedream to access your Shopify Partner Account, you will need to generate a new API key for Pipedream to use on your behalf.
+
+Include these scopes:
+
+- `View financials`
+- `Manage apps`
+
+## Using Shopify Partner Sources
```

Comment:
Were you still planning to finish these sections or do you want to remove for now?

Reply:
Added some docs, even though users won't see it.